dll/win32/newdev/lang/pl-PL.rc

```diff
@@ -101,4 +101,7 @@ BEGIN
     IDS_CHSOURCE_TITLE "Wybierz opcje wyszukiwania i instalacji"
     IDS_SEARCHDRV_TITLE "Szukanie..."
     IDS_INSTALLDRV_TITLE "Instalowanie..."
+
+    /* For IDD_BROWSE_FOR_FOLDER */
+    IDS_BROWSE_FOR_FOLDER_TITLE "Select the folder containing the drivers for the hardware."
```

Comment:
```suggestion
    IDS_BROWSE_FOR_FOLDER_TITLE "Wybierz folder zawierający sterowniki dla sprzętu."
```

@CC: @reactos/lang-polish

Reply:
After this PR is merged, you can open a new PULL REQUEST with the translation in your language. 
I don't know Polish